en/docs/insights/choreo-insights-api.md

```diff
@@ -0,0 +1,4217 @@
+<td>
+
+Start date of the time range. This 'from' date must be a date before the date defined in 'to'. If the defined date
+time entry does not have a timezone, UTC time zone(z) will be used. Example value: '2021-08-16T12:30:00.000+05:30'.
+It is required to specify a value for this parameter.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>to</strong></td>
+<td valign="top"><a href="#string">String</a>!</td>
+<td>
+
+End date of the time range. If the defined date time entry does not have a timezone, UTC time zone(z) will be used.
+Example value: '2021-08-16T12:30:00.000+05:30'. It is required to specify a value for this parameter.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### TopAPIsByAlertCountFilter
+
+Filters top APIs by alert count results.
+
+<table>
+<thead>
+<tr>
+<th colspan="2" align="left">Field</th>
+<th align="left">Type</th>
+<th align="left">Description</th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td colspan="2" valign="top"><strong>paginationFilter</strong></td>
+<td valign="top"><a href="#paginationfilter">PaginationFilter</a></td>
+<td>
+
+The pagination filter. This is an optional parameter. Therefore, if there is no requirement to filter by this
+parameter, omit this parameter.
+
+</td>
+</tr>
+<tr>
+<td colspan="2" valign="top"><strong>searchFilter</strong></td>
+<td valign="top"><a href="#searchfilter">SearchFilter</a></td>
+<td>
+
+The search filter. This is an optional parameter. Therefore, if there is no requirement to filter by this parameter,
+omit this parameter.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+## Enums
+
+### ErrorCodeType
+
+Represents an error code type.
+
+<table>
+<thead>
+<th align="left">Value</th>
+<th align="left">Description</th>
+</thead>
+<tbody>
+<tr>
+<td valign="top"><strong>_4XX</strong></td>
+<td>
+
+Client side errors.
+
+</td>
+</tr>
+<tr>
+<td valign="top"><strong>_5XX</strong></td>
+<td>
+
+Server side errors.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+### ErrorType
+
+Represents an error type.
+
+<table>
+<thead>
+<th align="left">Value</th>
+<th align="left">Description</th>
+</thead>
+<tbody>
+<tr>
+<td valign="top"><strong>PROXY</strong></td>
+<td>
+
+Proxy related errors.
+
+</td>
+</tr>
+<tr>
+<td valign="top"><strong>TARGET</strong></td>
+<td>
+
+Target related errors.
+
+</td>
+</tr>
+</tbody>
+</table>
+
+## Scalars
+
+### Boolean
+
+The <code>Boolean</code> scalar type represents <code>true</code> or <code>false</code>.
+
+### Float
+
+The <code>Float</code> scalar type represents signed double-precision fractional values as specified by [IEEE 754](https://en.wikipedia.org/wiki/IEEE_floating_point).
+
+### ID
+
+The <code>ID</code> scalar type represents a unique identifier, often used to refetch an object or as key for a cache. The ID type appears in a JSON response as a String; however, it is not intended to be human-readable. When expected as an input type, any string (such as <code>"4"</code>) or integer (such as <code>4</code>) input value will be accepted as an ID.
```

Comment:
```suggestion
The `ID` scalar type represents a unique identifier, often used to re-fetch an object or as a key for a cache. The `ID` type appears in a JSON response as a string, but it is not intended to be human-readable. When expected as an input type, any string (such as <code>"4"</code>) or integer (such as <code>4</code>) input value is accepted as an ID.
```

Reply:
We need to keep this as it is.This section is fully automatically generated according to the spec of graphql.